### PR TITLE
[VEN-2724][VEN-2725]: Deployments of XVS and Bridge on OpSepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description
This PR contains the deployment of XVS and Bridge on OpSepolia.

Resolves VEN-2724 and VEN-2725
```
Deployed Addresses: 
    "XVS": "0x789482e37218f9b26d8D9115E356462fA9A37116",
    "XVSBridgeAdmin": "0x6bBcB95eCF9BEc9AE91d5Ad227783e3913145321",
    "XVSProxyOFTDest": "0x79a36dc9a43D05Db4747c59c02F48ed500e47dF1"
```

**Note:** Need to update package version of governance and oracles.
## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -18,4 +18,6 @@ DEPLOYER_PRIVATE_KEY=
 #ARCHIVE_NODE_opbnbmainnet=https://opbnb-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_arbitrumsepolia="https://sepolia-rollup.arbitrum.io/rpc"
 #ARCHIVE_NODE_arbitrumone="https://open-platform.nodereal.io/<YOUR_KEY_HERE>/arbitrum-nitro/"
-#ARCHIVE_NODE_zksyncsepolia=https://zksync-sepolia.g.alchemy.com/v2/<YOUR_KEY_HERE>
\ No newline at end of file
+#ARCHIVE_NODE_zksyncsepolia=https://zksync-sepolia.g.alchemy.com/v2/<YOUR_KEY_HERE>
+#ARCHIVE_NODE_opsepolia="https://sepolia.optimism.io"
+#ARCHIVE_NODE_opmainnet="https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>"
```
Comment (chechu) [inline .env.example:23]:
```suggestion
#ARCHIVE_NODE_opsepolia=https://sepolia.optimism.io
#ARCHIVE_NODE_opmainnet=https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
```

File: deployments/opsepolia_addresses.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "opsepolia",
+  "chainId": "11155420",
+  "addresses": {
+    "DefaultProxyAdmin": "0xa9aaf2A1cCf2C3a87997942abaA740887cC89241",
+    "XVS": "0x789482e37218f9b26d8D9115E356462fA9A37116",
+    "XVSBridgeAdmin": "0x6bBcB95eCF9BEc9AE91d5Ad227783e3913145321",
+    "XVSBridgeAdmin_Implementation": "0x6E1bb2dd25CcE31A57d27F941a74786c67e8C6e3",
```
Comment (chechu) [inline deployments/opsepolia_addresses.json:8]: This contract is not verified on https://sepolia-optimism.etherscan.io/

Comment (Debugger022) [reply deployments/opsepolia_addresses.json:8]: https://sepolia-optimistic.etherscan.io/address/0x6E1bb2dd25CcE31A57d27F941a74786c67e8C6e3#code

**Comment by GitGuru7** [comment]
@Debugger022 CI checks are failing, LGTM!

**Review by chechu — APPROVED** [review]

**Pull request merged**